src/pages/RequestCallPage.js

```diff
@@ -0,0 +1,176 @@
+import * as React from 'react';
+import {View, Text, TextInput} from 'react-native';
+import _ from 'underscore';
+import {withOnyx} from 'react-native-onyx';
+import PropTypes from 'prop-types';
+import Str from 'expensify-common/lib/str';
+import HeaderWithCloseButton from '../components/HeaderWithCloseButton';
+import Navigation from '../libs/Navigation/Navigation';
+import styles from '../styles/styles';
+import ScreenWrapper from '../components/ScreenWrapper';
+import withLocalize, {withLocalizePropTypes} from '../components/withLocalize';
+import ONYXKEYS from '../ONYXKEYS';
+import compose from '../libs/compose';
+import FullNameInputRow from '../components/FullNameInputRow';
+import Button from '../components/Button';
+import FixedFooter from '../components/FixedFooter';
+import CONST from '../CONST';
+import Growl from '../libs/Growl';
+import {requestConciergeDMCall} from '../libs/actions/Inbox';
+import {fetchOrCreateChatReport} from '../libs/actions/Report';
+
+const propTypes = {
+    ...withLocalizePropTypes,
+
+    /** The personal details of the person who is logged in */
+    myPersonalDetails: PropTypes.shape({
```

Comment:
Not sure if that's relevant but maybe checkout @Jag96's code for displaying phone numbers if that's the primary `displayName` of the user : 
https://github.com/Expensify/Expensify.cash/pull/3728